contrib/mesos/docs/ha.md

```diff
@@ -1,4 +1,25 @@
-## High Availability
+# High Availability
+
+Kubernetes on Mesos supports 2 HA modes:
+
+* [Hot-standby](#hot-standby)
+* [Cold-standby](#cold-standby)
+
+In the hot-standby mode, all master components (apiserver, controller manager, and scheduler) actively run 
```

Comment:
Good point. In the documentation, I'd like to point out the fact that both hot-standby and cold-standby are viable options *eventually*, but as you pointed out, I should stress the point that hot-standby is currently work-in-progress. Will make some changes here to make this point more clear.